spec.html

```diff
@@ -23072,7 +23284,380 @@ <h1>Example Cyclic Module Record Graphs</h1>
+
+          <emu-table id="table-module-graph-cycle-async-fields-1" caption="Module fields after the initial Evaluate() call">
+            <table>
+              <thead>
+                <tr>
+                  <th>Module</th>
+                  <th>[[DFSIndex]]</th>
+                  <th>[[DFSAncestorIndex]]</th>
+                  <th>[[Status]]</th>
+                  <th>[[AsyncEvaluation]]</th>
+                  <th>[[AsyncParentModules]]</th>
+                  <th>[[PendingAsyncDependencies]]</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <th>_A_</th>
+                  <td>0</td>
+                  <td>0</td>
+                  <td>~evaluating-async~</td>
+                  <td>*true*</td>
+                  <td>&laquo; &raquo;</td>
+                  <td>2 (_B_ and _C_)</td>
+                </tr>
+                <tr>
+                  <th>_B_</th>
+                  <td>1</td>
+                  <td>0</td>
+                  <td>~evaluating-async~</td>
+                  <td>*true*</td>
+                  <td>&laquo; _A_ &raquo;</td>
+                  <td>1 (_D_)</td>
+                </tr>
+                <tr>
+                  <th>_C_</th>
+                  <td>2</td>
+                  <td>0</td>
+                  <td>~evaluating-async~</td>
+                  <td>*true*</td>
+                  <td>&laquo; _A_ &raquo;</td>
+                  <td>2 (_D_ and _E_)</td>
+                </tr>
+                <tr>
+                  <th>_D_</th>
+                  <td>3</td>
+                  <td>0</td>
+                  <td>~evaluating-async~</td>
+                  <td>*true*</td>
+                  <td>&laquo; _B_, _C_ &raquo;</td>
+                  <td>0</td>
+                </tr>
+                <tr>
+                  <th>_E_</th>
+                  <td>4</td>
+                  <td>4</td>
+                  <td>~evaluating-async~</td>
+                  <td>*true*</td>
+                  <td>&laquo; _C_ &raquo;</td>
+                  <td>0</td>
+                </tr>
+              </tbody>
+            </table>
+          </emu-table>
+
+          <p>Let us assume that _E_ finishes executing first. When that happens, AsyncModuleExecutionFulfilled is called, _E_.[[Status]] is set to ~evaluated~ and _C_.[[PendingAsyncDependencies]] is decremented to become 1. The fields of the updated modules are as given in <emu-xref href="#table-module-graph-cycle-async-fields-2"></emu-xref>.</p>
```

Comment:
The example is confused. The first paragraph talks about E transitioning to evaluated, suggesting it's not async. But here it's being async evaluated. I guess the first paragraph is wrong?

Reply:
Yes, I think the previous paragraph was wrong and _E_ should have been ~evaluating-async~ at that point. It only transitions here.